reference/configuration/framework.rst

```diff
@@ -204,7 +204,7 @@ ide
 Symfony turns file paths seen in variable dumps and exception messages into
 links that open those files right inside your browser. If you prefer to open
 those files in your favorite IDE or text editor, set this option to any of the
-following values: ``phpstorm``, ``sublime``, ``textmate``, ``macvim`` and ``emacs``.
+following values: ``phpstorm``, ``sublime``, ``textmate``, ``macvim``, ``emacs`` and ``atom``.
```

Comment:
Nice! Thanks for your contribution. I just have a very minor comment. To make docs easier to read out of the symfony.com website, we limit the length of the lines of text to approximately 80 characters. So, could you please edit this change and move `and atom.` to a new line? Thanks!

-----

Note: since I don't know if you have much or little experience with GitHub, just in case you need it, you can change the pull request using GitHub interface by clicking on this icon:

![github-edit-changes](https://user-images.githubusercontent.com/73419/34879453-91550490-f7ad-11e7-8a07-ac5bcf42dad0.png)